src/controllers/base.ts

```diff
@@ -5,7 +5,7 @@ export abstract class BaseController {
 
   async handle(ctx: Context) {
 
-    if ((this as any)[ctx.method] !== "function") {
+    if ((this as any)[ctx.method] !== 'function') {
```

Comment:
what is this crazy shit? `(this as any)`

Reply:
`as` casts an object to something else. In this case I am checking if the subclass defines a method named `get` or `post` so I can avoid the `allow` array. I don't love the syntax but it solves a minor annoyance for me in api2.

If the method exists, we allow it.